OpenEdXMobile/build.gradle

```diff
@@ -181,7 +184,7 @@ dependencies {
     provided 'org.roboguice:roboblender:3.0.1'
 
     // For the optional Nullable annotation
-    compile 'com.android.support:support-annotations:23.3.0'
+    compile 'com.android.support:support-annotations:25.0.1'
```

Comment:
@miankhalid 
I second the roboleetric compatibility comment. 
https://github.com/robolectric/robolectric/issues/1053

Is this update required?